moon-store-mac-os/moon-store-mac-os/Helpers/Persistence/Models/RoleSwiftDataModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RoleModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/14/24.
+//
+
+import SwiftData
+
+@Model
+final class RoleSwiftDataModel {
```

Comment:
Why the class name is so verbose? you can update the name like this:

```suggestion
final class Role {
```